perf-runner/run-tests.mjs

```diff
@@ -0,0 +1,62 @@
+import { config, validateConfig } from './src/config.mjs';
```

Comment:
I think this is a good start for a `runner-local` package. Maybe take a look at what I did here and follow that pattern?  https://github.com/expressjs/perf-wg/blob/main/packages/runner-docker/index.mjs